resources/assets/components/utilities/ScholarshipCard/GalleryBlockSignup.js

```diff
@@ -107,17 +137,22 @@ const GalleryBlockSignup = ({ campaignId, path }) => {
           : 'Apply Now'
       }
       href={path}
-      onClick={handleScholarshipCardShareClick}
+      onClick={() => {
+        handleScholarshipCardShareClick();
+        handleScholarshipButtonAnalytic();
+      }}
```

Comment:
@Dinnall Per https://github.com/DoSomething/phoenix-next/pull/2448#discussion_r535346053 - I think what we should do here is something like the following:
```suggestion
      onClick={() => {
        campaignData && campaignData.signups.length ? handleScholarshipButtonAnalytic() : handleScholarshipCardShareClick();
      }}
```